lifecycle_test.go

```diff
@@ -167,7 +169,24 @@ func TestLifecycleFullFlow(t *testing.T) {
 		// Verify the artifacts were distributed correctly by the provided path mappings.
 		assertExpectedArtifacts(t, tests.SearchAllDevRepo, tests.GetExpectedLifecycleDistributedArtifacts())
 	*/
+}
+
+func TestCreateBundleWithoutSpec(t *testing.T) {
+	cleanCallback := initLifecycleTest(t, artifactoryLifecycleMinVersion)
+	defer cleanCallback()
+
+	lcManager := getLcServiceManager(t)
```

Comment:
Better to move this function call a few lines below, closer to where it is needed.